test/uk/gov/hmrc/merchandiseinbaggage/controllers/DeclarationControllerSpec.scala

```diff
@@ -67,7 +67,7 @@ class DeclarationControllerSpec extends BaseSpecWithApplication with CoreTestDat
         val eventualResult = controller.findBy(declaration.mibReference, declaration.eori)(getRequest)
 
         status(eventualResult) mustBe 200
-        contentAsJson(eventualResult) mustBe Json.toJson(FindByResponse(declaration.declarationId))
```

Comment:
so in the end we are re-wrapping declarationId in an other case class.....we need a better way I think

Reply:
we no longer need `FindByResponse`  -  i will remove it

Reply:
i was gone say...this will break contract as FE is expecting the whole declaration now...did you not ran contract tests???